assets/stylesheets/babble.scss

```diff
@@ -69,6 +69,46 @@
   }
 }
 
+.babble-menu-modals {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  z-index: 100;
+}
+
+.babble-modal {
+  background-color: black;
+  opacity: 0.8;
```

Comment:
I think we want to bump this up even a little bit more; the busy text in the background is a little distracting even at 80%

![screen shot 2016-04-09 at 8 20 22 pm](https://cloud.githubusercontent.com/assets/750477/14402662/8be1a346-fe90-11e5-987b-1557a492c669.png)